README.md

```diff
@@ -24,21 +24,48 @@ The following languages have been translated by machine translation and must be
 * Russian ( _русский язык_ )
 * Chinese ( _中文_ )
 
-# Installation :
+# Installation
 
-You need :
-* a linux machine (tested only on debian bullseye).
-* python3 and python3-pip.
-* recommended : python3-tornado ,python3-jinja2 ,python3-pillow ,python3-pycurl ,python3-babel ,python3-numpy ,python3-boto3
+These install instructions are constantly tested via CI/CD pipeline on Debian Bullseye and Ubuntu Focal.
 
-```
-sudo pip install motioneye
-sudo motioneye_init
-```
+1. Install **Python 3** and build dependencies
+    Here as example the commands for APT-based Linux distributions:
+
+    On **ARMv6 and ARMv7** systems:
+    ```sh
+    sudo apt update
+    sudo apt --no-install-recommends install ca-certificates curl python3 python3-distutils
+    ```
+
+    On all other architectures additional development headers are required:
+    ```sh
+    sudo apt update
+    sudo apt --no-install-recommends install ca-certificates curl python3 python3-dev libcurl4-openssl-dev gcc libssl-dev
+    ```
+
+2. Install the Python package manager `pip`
+    ```sh
+    curl -sSfL 'https://bootstrap.pypa.io/get-pip.py' -o get-pip.py
+    sudo python3 get-pip.py
+    rm get-pip.py
+    ```
+
+    On **ARMv6 and ARMv7** systems, additionally configure `pip` to use pre-compiled wheels from [piwheels](https://piwheels.org/):
+    ```sh
+    printf '%b' '[global]\nextra-index-url=https://www.piwheels.org/simple/\n' | sudo tee /etc/pip.conf > /dev/null
+    ```
+
+3. Install and setup **motionEye**
+    ```sh
+    sudo python3 -m pip install 'https://github.com/motioneye-project/motioneye/archive/dev.tar.gz'
+    sudo motioneye_init
+    ```
+    _NB: `motioneye_init` currently assumes either an APT- or RPM-based distribution with `systemd` as init system. For a manual setup, config and service files can be found here: <https://github.com/motioneye-project/motioneye/tree/dev/motioneye/extra> _
 
 # Upgrade
-```
+
+```sh
 sudo systemctl stop motioneye
-sudo pip install motioneye --upgrade
+sudo python3 -m pip install -U --force-reinstall --no-deps 'https://github.com/motioneye-project/motioneye/archive/dev.tar.gz'
```

Comment:
Let's keep this self-documenting give that pip has both **--upgrade** and **--user**. 
```suggestion
sudo python3 -m pip install --upgrade --force-reinstall --no-deps 'https://github.com/motioneye-project/motioneye/archive/dev.tar.gz'
```